clock.py

```diff
@@ -18,7 +19,8 @@ def access_stack_overflow_api():
     if stack_exchange_api.have_logged_in(12) is False:
         message = "You haven't logged in for at least " + str(delta_hours) + " hours! \n " + \
                   "Access stackoverflow.com to save your login streak"
-        print("ERROR!\n" + message)
+        logging.basicConfig(level=logging.ERROR)
+        logging.error("ERROR!\n" + message)
```

Comment:
I know that this was the message previously printed, but I would remove the hardcoding of `ERROR!\n`. Therefore, `logging.error(message)` should be enough.